ui/app/mirrors/edit/[mirrorId]/nomirror.tsx

```diff
@@ -0,0 +1,28 @@
+import { Button } from '@/lib/Button';
+import { Label } from '@/lib/Label';
+import Link from 'next/link';
+
+const NoMirror = () => {
+  return (
+    <div
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        rowGap: '1rem',
+        justifyContent: 'center',
+      }}
+    >
+      <Label variant='title2'>Oops! </Label>
+      <Label as='label' style={{ fontSize: 18 }}>
+        We were unable to fetch details of this mirror. Please confirm if this
```

Comment:
This component is just a backup. Noticed some old mirrors of mine were being redirected to QRep because their workflowIDs were not found